v21.2/create-role.md

```diff
@@ -45,20 +45,34 @@ Unless a role is a member of the admin role, additional [privileges](#parameters
 
 ## Parameters
 
-| Parameter | Description |
+<style>
```

Comment:
yay markdown is great :-D

Reply:
do we have a docs site issue filed for this?  looks like a one-off fix due to length of the phrase in the Parameter column right?

(I'm not asking for any change in this PR but I think we should have this solved in the site design b/c this is gonna be fragile)